src/hooks/useLoadClientScript.ts

```diff
@@ -38,44 +38,47 @@ export function useLoadClientScript(props: NostoScriptProps) {
       window.nostojs(api => api.setAutoLoad(false))
     }
 
-    // Load Nosto client script if not already loaded externally
-    if (!isNostoLoaded() && !shopifyMarkets) {
-      const urlPartial = `/include/${account}`
-      const script = createScriptElement(urlPartial)
-      script.onload = scriptOnload
-      document.body.appendChild(script)
-    }
+    if (!loadScript) {
+      setClientScriptLoadedState(true)
```

Comment:
I would call scriptOnLoad instead
it includes this line and also executes the nosto.reload side effect with the proper test guard